qiskit_metal/qlibrary/core/design_check.py

```diff
@@ -0,0 +1,83 @@
+### BASIC PROGRAM THAT LOOKS FOR OVERLAP BETWEEN QCOMPONENTS
+from numpy import size
+import geopandas as gpd
+import pandas as pd
+
+
+class QDesignCheck():
+    """QDesign_Check contains various design checks.
```

Comment:
Maybe a bit more of a description here would be helpful

Reply:
Expanded on this description a bit.